public/test/test-utils.tsx

```diff
@@ -74,7 +74,9 @@ const getWrapper = ({
       <Provider store={reduxStore}>
         <GrafanaContext.Provider value={context}>
           <PotentialRouter history={history}>
-            <ModalsContextProvider>{children}</ModalsContextProvider>
+            <LocationServiceProvider service={locationService}>
```

Comment:
Saw a location provider was added to AppWrapper recently, so pulled it into here as well for more accurate tests